app/src/main/res/layout/activity_main.xml

```diff
@@ -6,30 +6,83 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <LinearLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:gravity="center_horizontal"
         android:orientation="vertical"
         android:padding="@dimen/default_margin">
 
-        <com.google.android.material.switchmaterial.SwitchMaterial
-            android:id="@+id/sm_3d"
+        <androidx.constraintlayout.helper.widget.Flow
+            android:id="@+id/checkboxFlow"
+            android:layout_width="match_parent"
```

Comment:
Check all Constraint layout. You need to put `0dp` if you want size match parent
https://stackoverflow.com/questions/37603751/set-width-to-match-constraints-in-constraintlayout